app/graphql/types/image.rb

```diff
@@ -18,6 +12,12 @@ def original
     }
   end
 
+  field :views, [Types::ImageView],
+    null: false,
+    description: 'The various generated views of this image' do
+      argument :names, [String], required: false
+    end
```

Comment:
Let's keep it consistent to what we did for other code.

```suggestion
  field :views, [Types::ImageView], null: false do
    description 'The various generated views of this image.'
    argument :names, [String], required: false
  end
```

Reply:
Do I just put another comment here because this GLARINGLY LARGE SUGGEST TEXT seems to have been ignored :P

Reply:
I s2g I committed this wtf